### ISSUE TITLE
Linkify the top pages

### ISSUE THREAD

**Issue by kasbah** [plausible/analytics#91]
Hello, we just enabled our public dashboard at https://plausible.io/kitspace.org/ (looks great!)

For our use, I think it would be cool if people could click on the top pages: i.e. go "oh what's this, it seems to be very popular" and go check out the page on our site.

**Comment by tckb** [comment]
I think for the actual cloud version you use feedback here ? https://plausible.nolt.io/ @ukutaht

**Comment by kasbah** [comment]
Thanks @tckb,  I think this would need to be a feature in this here software first to be deployed there though?

**Comment by metmarkosaric** [comment]
Thanks for submitting this! Yeah, that's the way we've done it until now. We did discuss to move the feature requests / roadmap to here too to consolidate it all. Cloud and self-hosted are the same product so we shouldn't separate them. 

Even though Nolt (our roadmap provider) works great it doesn't allow us to remove their GA code (we've tried to ask them) so... we've recently moved our docs for the same reason so will probably do the same for the roadmap.

We just have to make the move official!
